pandas/tests/computation/test_eval.py

```diff
@@ -1625,7 +1641,26 @@ def eval(self, *args, **kwargs):
     def test_unary_functions(self):
         df = DataFrame({'a': np.random.randn(10)})
         a = df.a
-        for fn in self.unary_fns:
+        unary_functions = [x for x in self.unary_fns
+                           if x not in ('floor', 'ceil')]
+        for fn in unary_functions:
+            expr = "{0}(a)".format(fn)
+            got = self.eval(expr)
+            with np.errstate(all='ignore'):
+                expect = getattr(np, fn)(a)
+            tm.assert_series_equal(got, expect, check_names=False)
+
+    def test_floor_and_ceil_functions_raise_error(self, ne_lt_2_6_9):
+        for fn in ('floor', 'ceil'):
+            msg = "\"{0}\" is not a supported function".format(fn)
+            with pytest.raises(ValueError, match=msg):
+                expr = "{0}(100)".format(fn)
+                self.eval(expr)
+
+    def test_floor_and_ceil_functions_evaluate_expressions(self, ne_gt_2_6_9):
```

Comment:
can you parametetrize (note i don't think this test adds anything beyond the basic one)